### PR TITLE
Test fix version

### PR DESCRIPTION
**Pull request by carioca-au**

### CODE REVIEW

File: deafrica/monitoring/readme.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Deafrica Scripts
```
Comment (alexgleith) [inline deafrica/monitoring/readme.md]: What's this about?

File: deafrica/monitoring/s2_gap_report.py [hunk 2]
```diff
@@ -168,11 +169,21 @@ def generate_buckets_diff(
 )
 @update_stac
 @slack_url
+@click.option("--version", is_flag=True, default=False)
```
Comment (alexgleith) [inline deafrica/monitoring/s2_gap_report.py:172]: This should be added to all CLI tools if it's going to be added to one.

Comment (alexgleith) [reply deafrica/monitoring/s2_gap_report.py:172]: And if it is, it should be one of the imported ones.

**Review by alexgleith — CHANGES_REQUESTED** [review]
Not sure what that readme is doing. And having version as a CLI option for one CLI app isn't useful.

File: deafrica/monitoring/s2_gap_report.py (hunk 2 above)
Comment (carioca-au) [reply deafrica/monitoring/s2_gap_report.py:172]: I'll do that

**Review by alexgleith — APPROVED** [review]

**Pull request merged**